internal/exec/shell_utils.go

```diff
@@ -185,13 +185,19 @@ func execTerraformShellCommand(
 		}
 	}()
 
-	// Set the Terraform environment variables to reference the var file
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_plan=-var-file=%s", varFile))
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_apply=-var-file=%s", varFile))
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_refresh=-var-file=%s", varFile))
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_import=-var-file=%s", varFile))
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_destroy=-var-file=%s", varFile))
-	componentEnvList = append(componentEnvList, fmt.Sprintf("TF_CLI_ARGS_console=-var-file=%s", varFile))
+	// Define the Terraform commands that may use var-file configuration
+	tfCommands := []string{"plan", "apply", "refresh", "import", "destroy", "console"}
+
+	// Check for existing var-file arguments in TF_CLI environment variables
+	for _, cmd := range tfCommands {
+		envVar := fmt.Sprintf("TF_CLI_ARGS_%s", cmd)
+		existing := os.Getenv(envVar)
+		if existing != "" && strings.Contains(existing, "-var-file=") {
+			u.LogWarning(atmosConfig, "Found var-file in environment! This may be overwritten by Atmos")
+		}
+		// Set the Terraform environment variable to reference the var file
+		componentEnvList = append(componentEnvList, fmt.Sprintf("%s=-var-file=%s", envVar, varFile))
```

Comment:
We still haven't solved the underlying intent here which is that we should also allow the user to specify their own TF_CLI_ARGS, wich would be appending to the current value of the corresponding ENV. Now we squash whatever value was there. If the value is set to anything, not just a var file, we should warn that this could interfere with atmos behavior.

Reply:
`TF_CLI_ARGS_name` does already work with the current version of atmos. (However it's overwritten by Atmos values, if defined)
```
TF_CLI_ARGS_plan="-var='foo=bar'" atmos terraform plan station -s dev
...

│ Error: Value for undeclared variable
│
│ A variable named "foo" was assigned on the command line, but the root module does not declare a variable of that name. To use this value, add a "variable" block to
│ the configuration.
╵
exit status 1
```

Reply:
However, `TF_CLI_ARGS` (without _name) does not work in practice. The TF_CLI_ARGS is passed to all TF CLI commands -- `init`, `workspace`, `plan`. So if it's an invalid arg for 1 of the 3, then that command fails:
```
TF_CLI_ARGS="-var 'foo=bar'" atmos terraform plan station -s dev

Initializing the backend...

Initializing provider plugins...
- Reusing previous version of hashicorp/local from the dependency lock file
- Reusing previous version of hashicorp/http from the dependency lock file
- Using previously-installed hashicorp/local v2.5.2
- Using previously-installed hashicorp/http v3.4.5

Terraform has been successfully initialized!
Usage: terraform [global options] workspace select NAME

  Select a different Terraform workspace.

Options:

    -or-create=false    Create the Terraform workspace if it doesn't exist.
Error parsing command-line flags: flag provided but not defined: -var

Usage: terraform [global options] workspace new [OPTIONS] NAME

  Create a new Terraform workspace.

Options:

    -lock=false         Don't hold a state lock during the operation. This is
                        dangerous if others might concurrently run commands
                        against the same workspace.

    -lock-timeout=0s    Duration to retry a state lock.

    -state=path         Copy an existing state file into the new workspace.
Error parsing command-line flags: flag provided but not defined: -var

exit status 1
```

Reply:
Plus, any other ENV var does not work. For example, this doesnt do anything
```
TF_VAR_foo='bar' atmos terraform plan station -s dev
```

Reply:
@osterman what is the goal with this task? 
1. Do we simply want to warn if `TF_CLI_ARGS_name` is set (no new behavior)
2. Do we want to support `TF_CLI_ARGS` somehow?
3. Do we want to support other env vars, such as `TF_VAR_varname`?

Reply:
I am on my phone and TF_CLI_ARGS is just my short hand. I don't mean that literally.

I guess I have to take a closer look at the code. I don't understand where we preserve the current value while appending new values to it .